extensions/wikia/AchievementsII/specials/templates/SpecialCustomize.tmpl.php

```diff
@@ -1,180 +1,199 @@
+		<input type="hidden" name="editToken" value="<?= Sanitizer::encodeAttribute( $editToken ); ?>"/>
+		<h2><?= wfMessage( 'achievements-create-edit-plus-category-title' )->escaped(); ?></h2>
+		<?= wfMessage( 'achievements-create-edit-plus-category-content' )->parse(); ?>
 		<p class="input">
-			<label><?=wfMessage('achievements-customize-new-category-track')->escaped();?></label><input name="edit_plus_category_name" type="text"/>
-			<button type="submit"><?=wfMessage('achievements-create-edit-plus-category')->escaped();?></button>
+			<label><?= wfMessage( 'achievements-customize-new-category-track' )->escaped(); ?></label><input
+				name="edit_plus_category_name" type="text"/>
+			<button type="submit"><?= wfMessage( 'achievements-create-edit-plus-category' )->escaped(); ?></button>
 		</p>
 	</form>
 </div>
 
 <div style="padding-right:320px;">
 
-<?if($successMsg != null):?>
-	<div class="successbox"><strong><?= $successMsg ?></strong></div>
-<?endif;?>
-<?if($errorMsg != null):?>
-	<div class="errorbox"><strong><?= $errorMsg ?></strong></div>
-<?endif;?>
-<?
-global $wgCityId, $wgScriptPath, $wgExternalSharedDB, $wgJsMimeType, $wgAchievementsEditOnly;
+	<? if( $successMsg != null ): ?>
+		<div class="successbox"><strong><?= $successMsg ?></strong></div>
+	<? endif; ?>
+	<? if( $errorMsg != null ): ?>
+		<div class="errorbox"><strong><?= $errorMsg ?></strong></div>
+	<? endif; ?>
+	<?
+	global $wgCityId, $wgScriptPath, $wgExternalSharedDB, $wgJsMimeType, $wgAchievementsEditOnly;
 
-$tracks = array();
-foreach( $config->getInTrackStatic() as $badgeTypeId => $trackData ){
-	if ( !$config->shouldShow( $badgeTypeId ) ) {
-		continue;
-	}
-	if ( $badgeTypeId != BADGE_EDIT ) {
-		$tracks[$badgeTypeId] = $trackData;
-	}
-	else {
-		$tracks[$badgeTypeId] = $trackData;
-		$editPlusCategoryTracks = $config->getInTrackEditPlusCategory();
+	$tracks = array();
+	foreach( $config->getInTrackStatic() as $badgeTypeId => $trackData ) {
+		if( !$config->shouldShow( $badgeTypeId ) ) {
+			continue;
+		}
+		if( $badgeTypeId != BADGE_EDIT ) {
+			$tracks[ $badgeTypeId ] = $trackData;
+		} else {
+			$tracks[ $badgeTypeId ] = $trackData;
+			$editPlusCategoryTracks = $config->getInTrackEditPlusCategory();
 
-		if( $editPlusCategoryTracks ) {
-			foreach( $editPlusCategoryTracks as $editPlusCategoryTypeId => $editPlusCategoryData ) {
-				$tracks[$editPlusCategoryTypeId] = array( 'category' => $editPlusCategoryData['category'], 'enabled' => $editPlusCategoryData['enabled'], 'laps' => $tracks[$badgeTypeId]['laps'], 'infinite' => $tracks[$badgeTypeId]['infinite'] );
+			if( $editPlusCategoryTracks ) {
+				foreach( $editPlusCategoryTracks as $editPlusCategoryTypeId => $editPlusCategoryData ) {
+					$tracks[ $editPlusCategoryTypeId ] = array( 'category' => $editPlusCategoryData[ 'category' ], 'enabled' => $editPlusCategoryData[ 'enabled' ], 'laps' => $tracks[ $badgeTypeId ][ 'laps' ], 'infinite' => $tracks[ $badgeTypeId ][ 'infinite' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "enabled" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "category" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)